rudder-web/src/main/scala/bootstrap/liftweb/AppConfig.scala

```diff
@@ -1721,28 +1707,21 @@ object RudderConfig extends Loggable {
         , asyncDeploymentAgent
         , uuidGen
       )
-    , new CheckSystemGroups (
-          rudderDitImpl
-        , roLdap
-        , ldapEntityMapper
-        , groupLibReadWriteMutex
-        , woNodeGroupRepository
-        , uuidGen
-      )
```

Comment:
This one is removed because the migration to dynamic system groups happened in 2.10 (so it should have already be gone), and I feared it could clash with the new checks.